### PR TITLE
Update Set-ProcessMitigation.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Few examples for remove and Reset and also added description for the parameter -remove.

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 1]
```diff
@@ -54,6 +54,46 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+```
+PS C:\> Set-Processmitigation -System -Enable DEP
+```
+
+To disable mitigations, you can replace `-Enable` with `-Disable`. However, for app-level mitigations, this will force the mitigation to be disabled only for that app.
+
+If you need to restore the mitigation back to the system default, you need to include the `-Remove` cmdlet as well, as in the following example:
+
+```PowerShell
+PS C:\> Set-Processmitigation -system -Remove -Disable DEP
```
Comment (tseward) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:70]: Capitalize 'Mitigation' and -'System'

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 2]
```diff
@@ -54,6 +54,46 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+
+To Disable SEHOP Compenent at the system level
+
+```
+PS C:\> Set-ProcessMitigation system -disable SEHOP
+```
+
+### Example 6
+
+To reset MItigtaion at system level
```
Comment (tseward) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:91]: `I` should be lowercase.

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 3]
```diff
@@ -54,6 +54,46 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+```
+PS C:\> Set-ProcessMitigation system -disable SEHOP
+```
+
+### Example 6
+
+To reset MItigtaion at system level 
+
+```
+PS C:\> Set-ProcessMitigation -System -reset
```
Comment (tseward) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:94]: Capitalize -Reset

**Comment by get-itips** [comment]
@msbemba Please kindly see the format on pre-existent examples 1 and 2 and use that, there is no introductory text after "Example X" title.

File: docset/windows/processmitigations/Set-ProcessMitigation.md (hunk 1 above)
Comment (msbemba) [reply docset/windows/processmitigations/Set-ProcessMitigation.md:70]: Made the changes.

File: docset/windows/processmitigations/Set-ProcessMitigation.md (hunk 2 above)
Comment (msbemba) [reply docset/windows/processmitigations/Set-ProcessMitigation.md:91]: Made the changes.

File: docset/windows/processmitigations/Set-ProcessMitigation.md (hunk 3 above)
Comment (msbemba) [reply docset/windows/processmitigations/Set-ProcessMitigation.md:94]: Made the changes.

**Comment by msbemba** [comment]
@dariomws  @get-itips  and @tseward  Thank you for the input. I have made changes as per your recommendation. I'll keep this in mind going forward.

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 4]
```diff
@@ -54,6 +54,43 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+PS C:\> Set-ProcessMitigation -System -Remove -Disable DEP
+```
+
+### Example 4
+
+```
+PS C:\> Set-ProcessMitigation -System -Enable SEHOP
+```
+
+Enable SEHOP Compenent at the system level
```
Comment (get-itips) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:77]:
```suggestion
Enable SEHOP Component at the system level
```

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 5]
```diff
@@ -54,6 +54,43 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+
+### Example 5
+
+```
+PS C:\> Set-ProcessMitigation -System -Disable SEHOP
+```
+
+Disable SEHOP Compenent at the system level
```
Comment (get-itips) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:85]:
```suggestion
Disable SEHOP Component at the system level
```

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 6]
```diff
@@ -54,6 +54,43 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+
+### Example 6
+
+```
+PS C:\> Set-ProcessMitigation -System -Reset
+```
+Reset Mitigtaion at system level
```
Comment (get-itips) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:92]:
```suggestion
Reset Mitigation at system level 
```

File: docset/windows/processmitigations/Set-ProcessMitigation.md [hunk 7]
```diff
@@ -54,6 +54,43 @@ PS C:\> Set-ProcessMitigation -PolicyFilePath settings.xml
...
+```
+
+Disable SEHOP Component at the system level
+
+### Example 6
+
+```
+PS C:\> Set-ProcessMitigation -System -Reset
+```
+Reset Mitigation at system level
```
Comment (dariomws) [inline docset/windows/processmitigations/Set-ProcessMitigation.md:92]:
```suggestion
Reset Mitigation at the system level.
```

**Comment by e0i** [comment]
@JohanFreelancer9 Would you be able to help Sriraman identify the source of validation errors above? It looks like a formatting issue. Thanks.

**Comment by JohanFreelancer9** [comment]
@e0i Unfortunately I am not able to identify the source of these validation errors.

**Comment by get-itips** [comment]
Maybe it is asking for the PowerShell code block on the example.

**Comment by get-itips** [comment]
@msbemba try with

<br>
```powershell

</br>
</br>
```

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1906

**Pull request merged**